docs/SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you find a security vulnerability in the Cirrus CI platform (the backend, web interface, etc.), please **follow the steps below**.
+
+1. **Do NOT comment about the vulnerability publicly.**
+1. Please email [`hello@cirruslabs.org`](mailto:hello@cirruslabs.org) with the following format:
```

Comment:
```suggestion
2. Please email [`hello@cirruslabs.org`](mailto:hello@cirruslabs.org) with the following format:
```

Reply:
Usually markdown would allow this but the code block may mess it up so I'll apply these